### PR TITLE
#5506 [Manager] Ensure Paper elements get the correct status from Bac…

### PR DESCRIPTION
**Pull request by konekoya**
…kend APIs

### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/connector/startConnectorEpic.js [hunk 1]
```diff
@@ -73,15 +79,15 @@ const startConnector$ = (values) => {
...
         status: CELL_STATUS.running,
       });
       return actions.startConnector.success(normalizedData);
     }),
     startWith(actions.startConnector.request({ connectorId })),
     catchError((err) => {
-      options.paperApi.updateElement(params.id, {
-        status: CELL_STATUS.stopped,
+      options.paperApi?.updateElement(params.id, {
+        status: err?.data?.state?.toLowerCase() ?? previousStatus,
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/connector/startConnectorEpic.js:90]: `??` 這種寫法第一次看到 🤔

Comment (konekoya) [reply ohara-manager/client/src/store/epics/connector/startConnectorEpic.js:90]: Nullish coalescing operator :)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Comment (eechih) [reply ohara-manager/client/src/store/epics/connector/startConnectorEpic.js:90]: 👍👍

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
rety manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@jackyoh This PR is ready to merge

**Comment by jackyoh** [comment]
> @jackyoh This PR is ready to merge

OK, I am going to merge this PR. Thank you!

**Pull request merged**